apps/workers/assetPreprocessingWorker.ts

```diff
@@ -16,6 +22,8 @@ import {
   triggerSearchReindex,
 } from "@hoarder/shared/queues";
 
+import { storeScreenshot } from "./crawlerWorker";
```

Comment:
Not a big fan of cross worker imports. If you want to share that logic, move the parts you're interested in in the `assetDb` util class.